torch/onnx/symbolic_opset12.py

```diff
@@ -60,5 +60,10 @@ def nll_loss2d(g, self, target, weight, reduction, ignore_index):
     return nll_loss(g, self, target, weight, reduction, ignore_index)
 
 
+def celu(g, self, alpha):
+    alpha = sym_help._maybe_get_const(alpha, 'f')
```

Comment:
What if self is not float?

Reply:
is Cast node needed?

Reply:
You mentioned int/long is not supported in PyTorch.
But looks like double is supported in PyTorch. However, ONNX Celu does not support it.
So we need a cast for double inputs.
Pleas verify this for all input types in your tests, and add casts where needed.